src/language-server/auto-completer.ts

```diff
@@ -241,6 +245,31 @@ export default class AutoCompleter extends Handler {
     };
   }
 
+  private getSlotNameCompletions(document: Document, location: AttributeValue) {
+    const parent = location.element.parentNode;
+    if (!parent || !parent.tagName) {
+      return {isIncomplete: false, items: []};
+    }
+    const [parentDefn] = document.getFeatures({
+      kind: 'element',
+      id: parent.tagName,
+      imported: true,
+      externalPackages: true
+    });
+    if (!parentDefn) {
+      return {isIncomplete: false, items: []};
+    }
+    const items =
+        parentDefn.slots.map(sl => sl.name).filter(name => !!name).map(name => {
```

Comment:
Maybe also filter the already declared slots? E.g. all sibling slot names?

Reply:
Oh! Good idea

Reply:
Hm, wait, I'm pretty sure you can distribute multiple nodes to the same slot. This came up with the paper-toolbar migration.

Reply:
Yeah, distributing multiple nodes to the same slot is allowed: https://jsbin.com/tekodehixo/1/edit?html,js,output